codebase/analysis/src.bin/aacgm_v2/aacgm_test.1.0/test_aacgm.c

```diff
@@ -230,7 +232,7 @@ void next(void)
     scanf("%c", &ch);
   } while (ch != '\n');
 
-/*  printf("\n\n");*/
+//  printf("\n\n");
```

Comment:
Recommend going back to c-style commenting instead of using c++ style.

Reply:
Or we can just delete this comment.

Reply:
I don't see a reason to keep the comment in.  It can be added back later if needed.